### PR TITLE
{lib,system,vis}[GCCcore/12.2.0] LibTIFF v4.4.0, jbigkit v2.1, libdeflate v1.14

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3104.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/7c3284df3f96693c21f46bc0d417d481 for a full test report.

File: easybuild/easyconfigs/l/libdeflate/libdeflate-1.14-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+# Updated to 1.8
+# J. Sassmannshausen NHS/GSTT
+
+easyblock = 'ConfigureMake'
+
+name = 'libdeflate'
+version = '1.14'
```
Comment (branfosj) [inline easybuild/easyconfigs/l/libdeflate/libdeflate-1.14-GCCcore-12.2.0.eb:9]: 1.15: https://github.com/ebiggers/libdeflate/releases/tag/v1.15 ?

File: easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.4.0-GCCcore-12.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'LibTIFF'
+version = '4.4.0'
+
+homepage = 'https://libtiff.maptools.org/'
```
Comment (branfosj) [inline easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.4.0-GCCcore-12.2.0.eb:6]: This is not loading for me. How about https://libtiff.gitlab.io/libtiff/ instead?

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/l/libdeflate/libdeflate-1.14-GCCcore-12.2.0.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/l/libdeflate/libdeflate-1.14-GCCcore-12.2.0.eb:9]: See https://github.com/boegel/easybuild-easyconfigs/pull/90

File: easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.4.0-GCCcore-12.2.0.eb (hunk 2 above)
Comment (branfosj) [reply easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.4.0-GCCcore-12.2.0.eb:6]: See https://github.com/boegel/easybuild-easyconfigs/pull/90

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/2cb27ead4861fa1066e1f55346698d14 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 1 out of 3 (3 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04.2 LTS (Focal Fossa), x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/3ef3a9a69a3d770d12d7079d2584543f for a full test report.

File: easybuild/easyconfigs/l/libdeflate/libdeflate-1.15-GCCcore-12.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+description = """Heavily optimized library for DEFLATE/zlib/gzip compression and decompression."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+github_account = 'ebiggers'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['58b95040df7383dc0413defb700d9893c194732474283cc4c8f144b00a68154b']
+
+builddependencies = [('binutils', '2.39')]
```
Comment (branfosj) [inline easybuild/easyconfigs/l/libdeflate/libdeflate-1.15-GCCcore-12.2.0.eb:21]:
```suggestion
builddependencies = [
    ('binutils', '2.39'),
    ('CMake', '3.24.3'),
]
```

Sorry. My test system has `CMake` in the OS.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/5f7594c06b8c91707e696646d435c168 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04.2 LTS (Focal Fossa), x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/172e8f82d5e0fbe77943fb645ed9628e for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Review by branfosj — APPROVED** [review]

**Pull request merged**